src/elona/profile/profile_manager.cpp

```diff
@@ -41,16 +52,20 @@ ProfileManager& ProfileManager::instance()
 
 
 
-void ProfileManager::create(
-    const ProfileId& new_profile_id,
-    const ProfileId& base_profile_id)
+void ProfileManager::create(const ProfileId& new_profile_id)
 {
-    ELONA_LOG("profile") << "Create '" << new_profile_id << "' "
-                         << " from '" << base_profile_id << "'.";
+    ELONA_LOG("profile") << "Create '" << new_profile_id << "'.";
 
-    const auto from = get_dir_for(base_profile_id);
     const auto to = get_dir_for(new_profile_id);
-    filesystem::copy_recursively(from, to);
+    fs::create_directory(to);
+
+    auto script_dir = to / "script";
+    fs::create_directory(script_dir.native());
```

Comment:
As `boost::filesystem::create_directly()` can take `boost::filesystem::path` directly, this `native()` method is unnecessary.